be/src/io/fs/multi_table_pipe.h

```diff
@@ -74,7 +74,7 @@ class MultiTablePipe : public KafkaConsumerPipe {
     std::atomic<uint64_t> _unplanned_row_cnt {0}; // trigger plan request when exceed threshold
     std::atomic<uint64_t> _inflight_plan_cnt {0}; // how many plan fragment are executing?
     std::atomic<bool> _consume_finished {false};
-    std::shared_ptr<StreamLoadContext> _ctx;
+    StreamLoadContext* _ctx;
```

Comment:
Please add comment here to explain why using raw pointer here. Or other people may refactor code and using shared ptr again.

Reply:
modified